bluebutton-data-pipeline-rif-load/src/test/java/gov/hhs/cms/bluebutton/data/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -239,11 +248,41 @@ private static void assertAreInDatabase(LoadAppOptions options, EntityManagerFac
 					Object recordId = entityManagerFactory.getPersistenceUnitUtil().getIdentifier(record);
 					Object recordFromDb = entityManager.find(record.getClass(), recordId);
 					Assert.assertNotNull(recordFromDb);
+					assertAreInRifDataloadHistoryTable(entityManager, record, dataSetManifest);
```

Comment:
I wouldn't check this once per record, like you're doing here. Instead, call this method separately from `loadSample(...)`: once per `RifFile`.

Reply:
Will research...